WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -484,7 +484,7 @@ def execute( self ):
         self.cfg.append( '-ddd' )
 
       if numberOfProcessor:
-        self.cfg.append( '-o "/Resources/Computing/CEDefaults/NumberOfProcessors=%d"' % numberOfProcessor )
+        self.cfg.append( '-o "/Resources/Computing/CEDefaults/SiteProcessors=%d"' % numberOfProcessor )
```

Comment:
In multi-core case, "/Resources/Computing/CEDefaults/NumberOfProcessors=%d" has been used in PoolComputingElement to get the total processors which the current multi-core pilot holds and count current available cores of the pilot.  It is conflicted with "NumberOfProcessors" in "PilotCommand.py", which is to describe the total cores which the WN holds and not used in other places now.  Therefore, we choose to change the parameter in PilotCommand.py.